style.css

```diff
@@ -0,0 +1,103 @@
+body
+{
+    margin: 0;
+    padding: 0;
+    background: url(banner.jpg);
+    background-size: cover;
+    background-position: center;
+    font-family: sans-serif; 
+}
+.login_box
+{
+    width: 345px;
+    background: rgb(0,0,0,0.5);
+    color: #fff;
+    top: 50%;
+    left: 50%;
+    position: absolute;
+    transform: translate(-50%,-50%);
+    box-sizing: border-box;
+    padding: 70px 20px;
+}
+h1
+{
+    margin: 0;
+    padding: 15 0 15px;
+    text-align: center;
+    font-size: 30px;
+    font-family:cursive;
+    

```

Comment:
### It is bad practice to leave empty lines before finishing the selector style

Same problem at lines 37, 38, 39, 56, 57, 66, 78 and 93